dpks/quant_matrix.py

```diff
@@ -438,29 +419,45 @@ def compare_groups(
 
         return self
 
-    def impute(self, method: str, **kwargs: int) -> QuantMatrix:
+    def classify(
+        self,
+        classifier,
+        shap_algorithm: str = "auto",
+        scale: bool = True,
+        rfe_step: int = 1,
+        rfe_min_features_to_select: int = 1,
+    ) -> QuantMatrix:
+        le = LabelEncoder()
+        Y = le.fit_transform(self.quantitative_data.var["group"].values)
+        X = self.quantitative_data.X.copy().transpose()
+        X = np.nan_to_num(X, copy=True, nan=0.0)
+        if scale:
+            X = StandardScaler().fit_transform(X)
+        clf = Classifier(classifier=classifier, shap_algorithm=shap_algorithm)
+        clf.fit(X, Y)
+        self.quantitative_data.obs["SHAP"] = clf.feature_importances_
+        selector = clf.recursive_feature_elimination(
+            X, Y, min_features_to_select=rfe_min_features_to_select, step=rfe_step
+        )
+        self.quantitative_data.obs["RFE_rank"] = selector.ranking_
```

Comment:
Could this field name be changed to something like FeatureRank? That way it fits with the rest of the columns in the meta-data dataframe

Reply:
Fixed